Formula/itk.rb

```diff
@@ -87,6 +85,9 @@ def install
     system "cmake", "-S", ".", "-B", "build", *std_cmake_args, *args
     system "cmake", "--build", "build"
     system "cmake", "--install", "build"
+
+    # Remove the bundled JRE installed by SCIFIO ImageIO plugin
+    (lib/"jre").rmtree if OS.linux? || Hardware::CPU.intel?
```

Comment:
Installed by https://github.com/scifio/scifio-imageio/blob/master/src/CMakeLists.txt for Linux, x86_64 macOS, and Windows.

Should be able to pick up user's `JAVA_HOME` or `java` on `PATH` given https://github.com/scifio/scifio-imageio/blob/master/src/itkSCIFIOImageIO.cxx.in#L319-L333